src/Domain/FundRepository.php

```diff
@@ -161,10 +162,15 @@ protected function setAnyFundData(Fund $fund, array $fundData): Fund
 
     protected function getNewFund(array $fundData): Fund
     {
+        $currencyCode = $fundData['currencyCode'] ?? 'GBP';
+        $name = $fundData['name'] ?? '';
```

Comment:
same values as at will be set just a moment later when the execution reaches here https://github.com/thebiggive/matchbot/blob/7434d20702494dd1a91f0ff7e31d270e7d87abf1/src/Domain/FundRepository.php#L156-L157